rclip/main.py

```diff
@@ -77,6 +78,9 @@ def _index_files(self, filepaths: List[str], metas: List[ImageMeta]):
         filtered_paths.append(path)
       except PIL.UnidentifiedImageError:
         pass
+      except DecompressionBombError:
+        print(f"warning: image '{path}' is too large, skipping", file=sys.stderr)
+        return
```

Comment:
Thank you for the updates to the PR!

Should this be `pass` instead of `return` so rclip continuee looping over the `filepaths`?

```suggestion
        pass
```